includes/class-sensei.php

```diff
@@ -563,6 +564,19 @@ public function install() {
 
 	} // End install()
 
+	/**
+	 * Check for plugin updates.
+	 *
+	 * @since 1.12.3
+	 */
+	public function update() {
```

Comment:
Should we make this `@access private`?

Reply:
Done in 9df0dac75ffea9b794dabf49bd45e8c0f26db80e